contracts/Migrations.sol

```diff
@@ -0,0 +1,23 @@
+pragma solidity ^0.4.4;
```

Comment:
It's documented here: http://truffleframework.com/docs/getting_started/migrations . I haven't dived into it - for now it's just needed to deploy the solidity to `testrpc` and run the tests. It's not something that's needed for production deployment (although I'm guessing it can be used for production deployment aswell, particularly if there are non-trivial deployments that are needed, where you need to follow a certain order and initialize things with other thing's addresses and such)